int32 manual_driving_mode
---
int32 return